README.md

```diff
@@ -181,8 +181,7 @@ To change it:
 let g:go_highlight_functions = 1
 let g:go_highlight_methods = 1
 let g:go_highlight_fields = 1
-let g:go_highlight_structs = 1
-let g:go_highlight_interfaces = 1
+let g:go_highlight_types = 1
```

Comment:
I agree that is quite unfortunate. I don't see an efficient way to separate parsing of `type Name struct` from `type Name interface`. Let me think about it a bit more. Maybe it can be done using `syn-region` syntax. 

On the other hand, why do we need to distinguish these two cases?